database/bfgd/database.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2024 Hemi Labs, Inc.
+// Copyright (c) 2025 Hemi Labs, Inc.
```

Comment:
For legal reasons, I think it may be better if we use a range of copyright years here (and in the other files):
```suggestion
// Copyright (c) 2024-2025 Hemi Labs, Inc.
```